week_1/String/longest_common_prefix.js

```diff
@@ -0,0 +1,21 @@
+// Write a function to find the longest common prefix string amongst an array of strings.
+
+// If there is no common prefix, return an empty string "".
+/**
+ * @param {string[]} strs
```

Comment:
let's be a bit more descriptive than "strs"
"words that will be checked for a common prefix"